go/git/get.go

```diff
@@ -178,15 +187,31 @@ func getMetadataInner(ctx context.Context, g *libkb.GlobalContext, folder *keyba
 			return nil, fmt.Errorf("unrecognized variant of GitLocalMetadataVersioned: %#v", version)
 		}
 
+		// Load UPAKs to get the device name.
+		upak, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{Name: responseRepo.LastModifyingUsername})
+		var deviceName string
+		for _, deviceKey := range upak.Current.DeviceKeys {
+			if deviceKey.DeviceID.Eq(responseRepo.LastModifyingDeviceID) {
+				deviceName = deviceKey.DeviceDescription
+				break
+			}
+		}
+		if deviceName == "" {
+			return nil, fmt.Errorf("can't find device name for %s's device ID %s", upak.Current.Username, responseRepo.LastModifyingDeviceID)
```

Comment:
Do they want this to be fatal? Maybe they'd prefer empty, idk. Maybe it will never happen.

Reply:
As far as I can tell this should never happen, and so I want us to have to think about it if it turns out it does :)